dashboards/winservices/winservives.json

```diff
@@ -0,0 +1,583 @@
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Start Mode",
+          "layout": {
+            "column": 4,
+            "row": 23,
+            "width": 4,
+            "height": 5
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.pie"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": true
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric select uniqueCount(service_name) where host.hostname IN ({{host.hostname}})  and service_name IN ({{service_name}}) and display_name IN ({{display_name}}) and state IN ({{state}})  and start_mode IN ({{start}}) and  metricName = 'windows_service_state'  facet start_mode  limit max"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "State different from  running, stopped or paused",
+          "layout": {
+            "column": 8,
+            "row": 23,
+            "width": 5,
+            "height": 5
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": " select host.hostname, display_name, service_name, state   FROM (select latest(state) as 'state' FROM Metric WHERE  host.hostname IN ({{host.hostname}}) and service_name IN ({{service_name}}) and display_name IN ({{display_name}}) and start_mode IN ({{start}}) and metricName = 'windows_service_state'  FACET host.hostname, display_name, service_name limit max)  where  state != 'running' and state !='stopped' and state !='paused' limit max"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": [
+    {
+      "name": "`host.hostname`",
```

Comment:
@seemantk I tried the following:
```
      "name": "`host.hostname`",
      "name": "'host.hostname",
      "name": "host.hostname",
```

I am not able to solve the issue:
```
ERROR: The following errors occurred while validating: quickstarts/winservices/config.yml
- quickstartMetadata/dashboards/rawConfiguration/variables/name: `name` must be set to alphanumeric with underscores or wrapped in backticks
```


Any idea? am I looking to the wrong? thing

Reply:
I'm not quite sure why we require backticks for wrapping names that have periods (since it doesn't [look](https://docs-dev.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/) like that's a platform requirement). Presumably, New Relic doesn't know to use this as a single identifier vs. an nested one.

Either way, to resolve this you'll probably want to update _all_ instances of `host.hostname` to be wrapped in backticks. I noticed quite a few places in the dashboard JSON where it wasn't wrapped.